k9mail/src/main/java/com/fsck/k9/activity/MessageLoaderHelper.java

```diff
@@ -180,7 +176,9 @@ private void onLoadMessageFromDatabaseFinished() {
 
         callback.onMessageDataLoadFinished(localMessage);
 
-        if (localMessage.isBodyMissing()) {
+        boolean messageIsIncomplete =
```

Comment:
Tiny detail: should be `messageIncomplete` so a getter would be `isMessageIncomplete()`.